src/containers/pages/InterventionPlan/NewPlan/General/tests/index.test.tsx

```diff
@@ -322,21 +337,21 @@ describe('containers/pages/NewPlan', () => {
         .first()
         .props() as any).formik.values
     );
-
-    expect(mockedStore.dispatch).toHaveBeenLastCalledWith(addPlanDefinition(payload));
+    const payloadOfInterest = extractPlanRecordResponseFromPlanPayload(payload);
+    expect(mockedStore.dispatch).toHaveBeenLastCalledWith(fetchPlanRecords([payloadOfInterest]));
   });
 
   it('New plan in planning tool is NOT added to store if API status is NOT 200', async () => {
     fetch.mockReject(() => Promise.reject('API is down'));
 
-    const initialState = {};
+    const initialState = existingState;
```

Comment:
Why is this necessary?

Reply:
we need to mock the data in the store because we dont have any data in start and this throws `undefined` when retrieving plans record from `store`